src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php

```diff
@@ -50,21 +50,29 @@ public function __construct(string $dispatcherService = 'event_dispatcher', stri
     /**
      * @return $this
      */
-    public function setHotPathEvents(array $hotPathEvents, string $tagName = 'container.hot_path')
+    public function setHotPathEvents(array $hotPathEvents)
```

Comment:
BC break?

Reply:
As the class is not tagged `@final`, indeed

Reply:
hmm, actually no. Child classes can add optional arguments, so this is indeed not a BC break.

Reply:
right